https-for-custom-domains.rst

```diff
@@ -0,0 +1,76 @@
+.. post:: August 10, 2018
+   :tags: security, feature
+   :author: David
+   :location: SAN
+
+
+HTTPS for Custom Domains
+========================
+
+Read the Docs hosts documentation for over 80,000 open source projects
+and over 2,500 of those projects are hosted on their own individual domains.
+Documentation hosted on ``*.readthedocs.io`` has supported HTTPS for a number of years,
+but one of our most requested features was to make HTTPS on other domains easy.
+Today we are happy to announce that **Read the Docs supports HTTPS on custom domains**!
+
+Earlier this year, Cloudflare contacted us to support HTTPS
+for the thousands of open source documentation projects on their own domains.
+They generously provided us with their `SSL for SaaS`_ package
+to ease the integration on our side.
+
+.. _SSL for Saas: https://www.cloudflare.com/ssl-for-saas-providers/
+
+
+Why HTTPS is important
+----------------------
+
+HTTPS encrypts traffic between your computer and Read the Docs' servers
+and allows your browser to verify that the website you are talking to
+is the one that you requested.
+HTTPS ensures there is no man-in-the-middle snooping on your connection.
+
+
+Configuring your domain
+-----------------------
+
+If you followed our `documentation for custom domains`_
+and you are using a ``CNAME`` to point to Read the Docs,
+no action is required on your part.
+
+If you have had your Read the Docs setup for many years
+you may have followed some older directions.
+Depending on your setup, we may not have been able to issue a certificate for your domain.
```

Comment:
This is a big ambiguous. We should be explicit here with what a "wrong" config looks like.